adds citation file

neuromusic opened the pull request:
# Description

adds a `CITATION.cff`-formatted citation file to the napari repo. 

this standard format will enable

- Github to render "How to cite" on the sidebar
- Zenodo will read this file to populate authors, etc in the "official" archived version of each release

citation generated from current zenodo metadata, using the [doi2cff](https://doi2cff.readthedocs.io/en/latest/) command line utility

## Type of change

housekeeping/documentation

# References

closes #1004

# How has this been tested?
<!-- Please describe the tests that you ran to verify your changes. -->
- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).

neuromusic committed:
adds citation file

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+doi: 10.5281/zenodo.5399494
+# FIXME splitting of full names is error prone, please check if given/family name are correct
+authors:
+- given-names: Nicholas
```
@sofroniewn can you review your name & affiliation?

sofroniewn replied:
Name looks good, I suggested expanding CZI to Chan Zuckerberg Initiative

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+- given-names: Nicholas
+  family-names: Sofroniew
+  affiliation: CZI
+- given-names: Talley
```
@tlambert03 can you review your name & affiliation?

neuromusic replied:
thanks!

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+- given-names: Talley
+  family-names: Lambert
+  affiliation: Harvard University
+- given-names: Kira
```
@kne42 can you review your name & affiliation?

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+- given-names: Kira
+  family-names: Evans
+  affiliation: '@chanzuckerberg'
+- given-names: Juan
```
@jni can you review your name & affiliation?

jni replied:
It's good!

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+- given-names: Juan
+  family-names: Nunez-Iglesias
+  affiliation: Biomedicine Discovery Institute, Monash University
+- given-names: Grzegorz
```
@Czaki can you review your name & affiliation?

neuromusic replied:
thanks!

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+- given-names: Grzegorz
+  family-names: Bokota
+  affiliation: University of Warszaw, CeNT
+- given-names: Philip
```
@pwinston can you review your name & affiliation?

pwinston replied:
- given-names: Philip
  family-names: Winston
  affiliation: Tobeva Software

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+  affiliation: University of Warszaw, CeNT
+- given-names: Philip
+  family-names: Winston
+- given-names: Gonzalo
```
@goanpeca can you review your name & affiliation?

goanpeca replied:
Yes sir!

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+- given-names: Gonzalo
+  family-names: Peña-Castellanos
+  affiliation: Quansight, Inc. / SIHSA Ltda
+- given-names: Kevin
```
@kevinyamauchi can you review your name & affiliation?

kevinyamauchi replied:
Looks good to me! Thanks, @neuromusic !

neuromusic replied:
thank you!

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+- given-names: Kevin
+  family-names: Yamauchi
+  affiliation: Iber Lab - ETH Zürich
+- given-names: Matthias
```
@Carreau can you review your name & affiliation?

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+- given-names: Matthias
+  family-names: Bussonnier
+  affiliation: '@Quansight-Labs'
+- given-names: Draga
```
@DragaDoncila can you review your name & affiliation?

DragaDoncila replied:
@neuromusic looks fine, just tweaked the family names - had to look up what a `name-particle` was so TIL!

neuromusic replied:
oh wow, fascinating! yeah, this was what the `doi2cff` tool generated from the zenodo entry

neuromusic committed:
corrects name and license

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+- given-names: Draga
+  family-names: Pop
+  name-particle: Doncila
+- given-names: ACS
```
@AhmetCanSolak can you review your name & affiliation?

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+- given-names: ACS
+  family-names: ''
+  affiliation: Chan Zuckerberg Biohub
+- given-names: ziyangczi
```
@potating-potato can you review your name & affiliation?

liu-ziyang replied:
Given name: Ziyang
Family name: Liu
Affiliation: Chan Zuckerberg Initiative Foundation

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+- given-names: ziyangczi
+  family-names: ''
+  affiliation: '@chanzuckerberg'
+- given-names: Pam
```
@ppwadhwa can you review your name & affiliation?

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+  affiliation: '@chanzuckerberg'
+- given-names: Pam
+  family-names: ''
+- given-names: alisterburt
```
@alisterburt can you review your name & affiliation?

neuromusic replied:
thanks!

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+  family-names: ''
+- given-names: alisterburt
+  family-names: ''
+- given-names: Genevieve
```
@GenevieveBuckley can you review your name & affiliation?

GenevieveBuckley replied:
Looks good to me! 
While technically Dask is my affiliation for 11 months in 2021, I was at Monash before and after that, and for longer periods of time - so I think this is fine.

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+- given-names: Genevieve
+  family-names: Buckley
+  affiliation: Monash University
+- given-names: Andy
```
@andy-sweet can you review your name & affiliation?

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+  affiliation: Monash University
+- given-names: Andy
+  family-names: Sweet
+- given-names: Lukasz
```
@lukasz-migas can you review your name & affiliation?

lukasz-migas replied:
That looks good to me! 👍

neuromusic replied:
awesome! thanks!

sofroniewn commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+authors:
+- given-names: Nicholas
+  family-names: Sofroniew
+  affiliation: CZI
```
```suggestion
  affiliation: Chan Zuckerberg Initiative
```

DragaDoncila commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+  affiliation: '@Quansight-Labs'
+- given-names: Draga
+  family-names: Pop
+  name-particle: Doncila
```
```suggestion
- given-names: Draga
  family-names: Doncila Pop
```

Czaki commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+  affiliation: Biomedicine Discovery Institute, Monash University
+- given-names: Grzegorz
+  family-names: Bokota
+  affiliation: University of Warszaw, CeNT
```
```suggestion
  affiliation: University of Warsaw, Faculty of Mathematics, Informatics, and Mechanics
```

alisterburt commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+- given-names: Pam
+  family-names: ''
+- given-names: alisterburt
+  family-names: ''
```
```suggestion
- given-names: Alister
  family-names: Burt
  affiliation: MRC-LMB
```

alisterburt replied:
thanks @neuromusic - hope this looks right!

neuromusic committed:
expands "CZI"

Co-authored-by: Nicholas Sofroniew <sofroniewn@gmail.com>

neuromusic committed:
fixes Draga's family names

Co-authored-by: Draga Doncila Pop <17995243+DragaDoncila@users.noreply.github.com>

tlambert03 commented on CITATION.cff:
```diff
@@ -0,0 +1,95 @@
+  affiliation: Chan Zuckerberg Initiative
+- given-names: Talley
+  family-names: Lambert
+  affiliation: Harvard University
```
```suggestion
  affiliation: Harvard Medical School
```

neuromusic committed:
fixes Grzegorz's affiliation

Co-authored-by: Grzegorz Bokota <bokota+github@gmail.com>

neuromusic committed:
fixes Alister's name & affiliation

Co-authored-by: alisterburt <alisterburt@gmail.com>

neuromusic committed:
fixes Talley's affiliation

Co-authored-by: Talley Lambert <talley.lambert@gmail.com>

andy-sweet commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+  family-names: Buckley
+  affiliation: Monash University
+- given-names: Andy
+  family-names: Sweet
```
```suggestion
- given-names: Andrew
  family-names: Sweet
  affiliation: Chan Zuckerberg Initiative
```

Probably makes sense to be consistent about the CZI affiliation across all contributors. Probably Chan Zuckerberg Initiative like @sofroniewn suggested, or maybe CZI or @chanzuckerberg.

liu-ziyang commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+  affiliation: Chan Zuckerberg Biohub
+- given-names: ziyangczi
+  family-names: ''
+  affiliation: '@chanzuckerberg'
```
```suggestion
- given-names: Ziyang
  family-names: Liu
  affiliation: Chan Zuckerberg Initiative Foundation
```

Czaki commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+- given-names: Juan
+  family-names: Nunez-Iglesias
+  affiliation: Biomedicine Discovery Institute, Monash University
+- given-names: Grzegorz
+  family-names: Bokota
+  affiliation: University of Warsaw, Faculty of Mathematics, Informatics, and Mechanics
```
```suggestion
- given-names: Grzegorz
  family-names: Bokota
  affiliation: University of Warsaw, Faculty of Mathematics, Informatics, and Mechanics
  orcid: https://orcid.org/0000-0002-5470-1676
```

tlambert03 commented on CITATION.cff:
```diff
@@ -0,0 +1,96 @@
+- given-names: Abigail
+  family-names: McGovern
+  affiliation: Monash University
+version: 0.4.11
```
i assume this is something we need to manually bump?

neuromusic replied:
yes. 

looks like it could be dropped? https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md

or is there other tooling the project is using to keep version info sync'd?

neuromusic replied:
dropped the version info & switched to zenodo's "concept" DOI so that this doesn't need to be kept up to date

neuromusic committed:
adds ORCID for Grzegorz

Co-authored-by: Grzegorz Bokota <bokota+github@gmail.com>

neuromusic committed:
add ziyang's info

Co-authored-by: Ziyang Liu <zliu@chanzuckerberg.com>

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,97 @@
+- given-names: Lukasz
+  family-names: Migas
+  affiliation: Delft University of Technology
+- given-names: Volker
```
@VolkerH can you review your name & affiliation?

VolkerH replied:
The name "Volker Hilsenstein" is correct.
Current affiliation: 
EMBL Heidelberg, Germany

Thanks Justin!

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,97 @@
+  affiliation: Delft University of Technology
+- given-names: Volker
+  family-names: Hilsenstein
+- given-names: Lorenzo
```
@brisvag can you review your name & affiliation?

brisvag replied:
Done in a separate comment.

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,97 @@
+- given-names: Lorenzo
+  family-names: Gaifas
+  affiliation: '@gutsche-lab'
+- given-names: Jordão
```
@JoOkuma can you review your name & affiliation?

JoOkuma replied:
It's correct.

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,97 @@
+- given-names: Jordão
+  family-names: Bragantini
+  affiliation: Chan Zuckerberg Biohub
+- given-names: Jaime
```
@jaimergp can you review your name & affiliation?

jaimergp replied:
LGTM! Thanks!

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,97 @@
+- given-names: Jaime
+  family-names: Rodríguez-Guerra
+  affiliation: '@quansight'
+- given-names: Hector
```
@hectormz

hectormz replied:
added a new comment for suggested change!

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,97 @@
+- given-names: Hector
+  family-names: ''
+  affiliation: AMDI
+- given-names: Jeremy
```
@freeman-lab can you review your name & affiliation?

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,97 @@
+- given-names: Jeremy
+  family-names: Freeman
+  affiliation: '@carbonplan'
+- given-names: Peter
```
@boonepeter can you review your name & affiliation?

boonepeter replied:
Looks good thanks!

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,97 @@
+  affiliation: '@carbonplan'
+- given-names: Peter
+  family-names: Boone
+- given-names: Alan
```
@quantumjot can you review your name & affiliation?

quantumjot replied:
Looks good.

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,97 @@
+  family-names: Lowe
+  name-particle: R
+  affiliation: UCL & The Alan Turing Institute
+- given-names: Christoph
```
@cgohlke can you review your name & affiliation?

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,97 @@
+- given-names: Christoph
+  family-names: Gohlke
+  affiliation: University of California, Irvine
+- given-names: Loic
```
@royerloic can you review your name & affiliation?

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,97 @@
+- given-names: Loic
+  family-names: Royer
+  affiliation: Chan Zuckerberg Biohub
+- given-names: Andrea
```
@kir0ul can you review your name & affiliation?

kir0ul replied:
It would be:
```yaml
- given-names: Andrea
  family-names: Pierré
  affiliation: Brown University
```
Thanks @neuromusic!

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,97 @@
+  affiliation: Chan Zuckerberg Biohub
+- given-names: Andrea
+  family-names: PIERRÉ
+- given-names: Hagai
```
@HagaiHargil can you review your name & affiliation?

neuromusic commented on CITATION.cff:
```diff
@@ -0,0 +1,97 @@
+  family-names: PIERRÉ
+- given-names: Hagai
+  family-names: Har-Gil
+- given-names: Abigail
```
@AbigailMcGovern can you review your name & affiliation?

brisvag commented on CITATION.cff:
```diff
@@ -0,0 +1,97 @@
+  family-names: Hilsenstein
+- given-names: Lorenzo
+  family-names: Gaifas
+  affiliation: '@gutsche-lab'
```
Your comment was a bit off-line, so I'm making a new one:
```suggestion
- given-names: Lorenzo
  family-names: Gaifas
  affiliation: Gutsche Lab - University of Grenoble
```

hectormz commented on CITATION.cff:
```diff
@@ -0,0 +1,97 @@
+  affiliation: '@quansight'
+- given-names: Hector
+  family-names: ''
+  affiliation: AMDI
```
```suggestion
- given-names: Hector
  family-names: Muñoz
  affiliation: University of California, Los Angeles
```

HagaiHargil commented on CITATION.cff:
```diff
@@ -0,0 +1,97 @@
+- given-names: Andrea
+  family-names: PIERRÉ
+- given-names: Hagai
+  family-names: Har-Gil
```
```suggestion
  family-names: Har-Gil
  affiliation: Tel Aviv University, Israel
```

tlambert03 commented on CITATION.cff:
```diff
@@ -0,0 +1,97 @@
+  affiliation: Chan Zuckerberg Initiative
+- given-names: Talley
+  family-names: Lambert
+  affiliation: Harvard Medical School
```
```suggestion
  affiliation: Harvard Medical School
  orcid: "https://orcid.org/0000-0002-2409-0181"
```

😛

GenevieveBuckley commented on CITATION.cff:
```diff
@@ -0,0 +1,97 @@
+  affiliation: MRC-LMB
+- given-names: Genevieve
+  family-names: Buckley
+  affiliation: Monash University
```
```suggestion
  affiliation: Monash University
  orcid:  https://orcid.org/0000-0003-2763-492X 
```

GenevieveBuckley replied:
Turns out I have an ORCID and used it exactly once in my life

neuromusic committed:
adds genevieve's ORCID

Co-authored-by: Genevieve Buckley <30920819+GenevieveBuckley@users.noreply.github.com>

neuromusic committed:
adds talley's ORCID

Co-authored-by: Talley Lambert <talley.lambert@gmail.com>

neuromusic committed:
fixes lorenzo's affiliation

Co-authored-by: Lorenzo Gaifas <brisvag@gmail.com>

neuromusic committed:
fixes hector's name & affiliation

Co-authored-by: Hector <23343812+hectormz@users.noreply.github.com>

neuromusic committed:
adds Hagai's affilation

Co-authored-by: Hagai Har-Gil <hagaihargil@gmail.com>

sofroniewn commented on CITATION.cff:
```diff
@@ -0,0 +1,100 @@
+authors:
+- given-names: Nicholas
+  family-names: Sofroniew
+  affiliation: Chan Zuckerberg Initiative
```
```suggestion
  affiliation: Chan Zuckerberg Initiative
  orcid: "https://orcid.org/0000-0002-3426-0914"
```

hectormz commented on CITATION.cff:
```diff
@@ -0,0 +1,100 @@
+- given-names: Jaime
+  family-names: Rodríguez-Guerra
+  affiliation: '@quansight'
+- given-names: Hector
+  family-names: Muñoz
+  affiliation: University of California, Los Angeles
```
```suggestion
- given-names: Hector
  family-names: Muñoz
  affiliation: University of California, Los Angeles
  orcid: https://orcid.org/0000-0001-7851-2549
```

brisvag commented on CITATION.cff:
```diff
@@ -0,0 +1,100 @@
+  affiliation: Delft University of Technology
+- given-names: Volker
+  family-names: Hilsenstein
+- given-names: Lorenzo
+  family-names: Gaifas
+  affiliation: Gutsche Lab - University of Grenoble
```
```suggestion
- given-names: Lorenzo
  family-names: Gaifas
  affiliation: Gutsche Lab - University of Grenoble
  orcid: https://orcid.org/0000-0003-4875-9422
```

neuromusic committed:
adds nick's affiliation

Co-authored-by: Nicholas Sofroniew <sofroniewn@gmail.com>

neuromusic committed:
Adds affiliation and ORCID info

Co-authored-by: Andy Sweet <andrew.d.sweet@gmail.com>
Co-authored-by: Lorenzo Gaifas <brisvag@gmail.com>
Co-authored-by: Hector <23343812+hectormz@users.noreply.github.com>

Carreau commented on CITATION.cff:
```diff
@@ -0,0 +1,104 @@
+  affiliation: Iber Lab - ETH Zürich
+- given-names: Matthias
+  family-names: Bussonnier
+  affiliation: '@Quansight-Labs'
```
```suggestion
  affiliation: '@Quansight-Labs'
  orcid: "http://orcid.org/0000-0002-7636-8632"
```

neuromusic committed:
future-proof for zenodo integration

- drop version (so that zenodo assigns)
- change DOI to zenodo's "concept" DOI
- drop "date released" b/c not pinned to version

neuromusic committed:
adds stray affiliations

neuromusic committed:
adds Volker's ORCID

neuromusic committed:
fixing a few affiliations

neuromusic committed:
Update CITATION.cff

Co-authored-by: Matthias Bussonnier <bussonniermatthias@gmail.com>

sofroniewn commented on CITATION.cff:
```diff
@@ -0,0 +1,105 @@
+# YAML 1.2
+# Metadata for citation of this software according to the CFF format (https://citation-file-format.github.io/)
+cff-version: 1.0.3
+message: If you use this software, please cite it using these metadata.
```
```suggestion
message: If you use this software, please cite it using these metadata
```

sofroniewn reviewed (commented):
Trying to trigger a re-run of the manifest check

sofroniewn committed:
Update CITATION.cff

sofroniewn commented on CITATION.cff:
```diff
@@ -0,0 +1,105 @@
+# YAML 1.2
+# Metadata for citation of this software according to the CFF format (https://citation-file-format.github.io/)
+cff-version: 1.0.3
+message: If you use this software, please cite it using these metadata
```
```suggestion
message: If you use this software, please cite it using these metadata.
```

sofroniewn committed:
Update CITATION.cff

sofroniewn committed:
Update MANIFEST.in

sofroniewn committed:
Merge branch 'main' into jk/citation.cff

jni committed:
Merge branch 'main' into jk/citation.cff
